orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/config/PreconfiguredWebhookProperties.groovy

```diff
@@ -67,6 +69,17 @@ class PreconfiguredWebhookProperties {
     }
   }
 
+  static class WebhookParameter {
+    String label
+    String defaultValue
+    String value
+    ParameterType type = string
+
+    static enum ParameterType {
```

Comment:
@ajordens i added this here. if you can think of a better way to do this, let me know.